### ISSUE TITLE
packages installation

### ISSUE THREAD

**Issue by manolis-pi** [NoTengoBattery/openwrt#12]
Hi,
I would like  to install some additional packages, not available in this release.
Will it be possible to use command line opkg and in that case what will be the URL to be used with opkg?

Thanks

**Comment by NoTengoBattery** [comment]
Hello!

The first thing that I may ask you is your device, the release version you are using, and the missing package. In general, downloading the `ipk` from the [OpenWrt Snapshots](https://downloads.openwrt.org/snapshots/packages/) and uploading it using LuCI should work. Since this is the IPQ40xx version, the architecture is: `arm_cortex-a7_neon-vfpv4`.

Anyway, with enough information, I can point you out to the direct download link[s] to try. I would like to know a little more information about your particular issue to provide a proper solution and add the missing package to the future release. Note that if the package needs an extra kernel module not available, it will be impossible to install and should be added to the next release.

Please let me know if you need more help. Thanks!

**Comment by manolis-pi** [comment]
Thanks for the prompt reply! 
I had already tried with wifitoggle (exactly the same way you suggested) and  it succeeded, I was just not sure if I had done it the right way or I broke anything without knowing it.
My device is Linksys EA6350 v3 and I am using the latest version: 2.10
I can continue downloading the rest of the packages, just for your information and if you would like to include them in the next release they are the following:

1. wifitoggle (on/off switch using the wps button for the wifi)
2. ddns-scripts_no-ip_com (ddns package for no-ip)
3. 
prometheus-node-exporter-lua
prometheus-node-exporter-lua-nat_traffic
prometheus-node-exporter-lua-netstat
prometheus-node-exporter-lua-openwrt
prometheus-node-exporter-lua-wifi
prometheus-node-exporter-lua-wifi_stations

node metrics exporter for Prometheus monitoring tool
